lib/runtime/interfaces.go

```diff
@@ -11,28 +11,34 @@ import (
 
 // Storage runtime interface.
 type Storage interface {
+	// Main trie
```

Comment:
I would suggest to split them into specific interfaces that composes the storage like `Trie`, `ChildTrie`, `Transactional`, `Runtime`

Reply:
Done